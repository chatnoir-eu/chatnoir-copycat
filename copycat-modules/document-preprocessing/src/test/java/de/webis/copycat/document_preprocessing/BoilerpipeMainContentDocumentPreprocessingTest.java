package de.webis.copycat.document_preprocessing;

import org.approvaltests.Approvals;
import org.junit.Test;

public class BoilerpipeMainContentDocumentPreprocessingTest extends DocumentPreprocessingTest {

	public BoilerpipeMainContentDocumentPreprocessingTest() {
		super(new BoilerpipeDocumentPreprocessing());
	}

	@Test
	public void approveWebisHtmlPage() {
		Approvals.verify(preprocessedHtml("webis-de"));
	}
	
	@Test
	public void approveToucheHtmlPage() {
		Approvals.verify(preprocessedHtml("touche-webis-de"));
	}
	
	@Test
	public void approveSigirHtmlPage() {
		Approvals.verify(preprocessedHtml("sigir-org-2021-resource"));		
	}
	

}
