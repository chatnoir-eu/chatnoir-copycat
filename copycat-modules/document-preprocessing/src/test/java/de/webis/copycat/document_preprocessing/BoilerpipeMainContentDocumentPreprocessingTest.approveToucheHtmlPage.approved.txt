important dates synopsis call decision making processes be it at the societal or at the personal level often come to a point where one side challenges the other with a why question which is a prompt to justify some stance based on arguments since technologies for argument mining are maturing at a rapid pace also ad hoc argument retrieval becomes a feasible task in reach we invite you to participate in the second touché lab on argument retrieval at clef 2021 featuring two tasks details on the touché 2020 edition can be found in the overview paper or on last year lab web page shared tasks